docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,533 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "94f82b0d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "issue_to_visualize = issue_idx[1]\n",
+    "label = labels[issue_to_visualize]\n",
+    "prediction = predictions[issue_to_visualize]\n",
+    "score = scores[issue_to_visualize]\n",
+    "\n",
+    "image_path = IMAGE_PATH + label['seg_map']\n",
+    "print(image_path, '| idx', issue_to_visualize , '| label quality score:', score, '| is issue: True')\n",
+    "visualize(image_path, label=label, prediction=prediction, class_names=class_names, overlay=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5532f1e1",
+   "metadata": {},
+   "source": [
+    "The monitor in this image is missing annotations for a cup and a person -- who is working on a chair in the background. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c62dac6f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "issue_to_visualize = issue_idx[2]\n",
+    "label = labels[issue_to_visualize]\n",
+    "prediction = predictions[issue_to_visualize]\n",
+    "score = scores[issue_to_visualize]\n",
+    "\n",
+    "image_path = IMAGE_PATH + label['seg_map']\n",
+    "print(image_path, '| idx', issue_to_visualize , '| label quality score:', score, '| is issue: True')\n",
+    "visualize(image_path, label=label, prediction=prediction, class_names=class_names, overlay=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1549af39",
+   "metadata": {},
+   "source": [
+    "The birthday party in this image should have had individual bounding boxes around each kid (the COCO guidelines state only groups with 10+ objects of the same type can be a \\\"crowd\\\" bounded by a single box). \n",
+    "\n",
+    "All of these examples received low label quality scores reflecting their low annotation quality in the original dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03d5a521",
+   "metadata": {},
+   "source": [
+    "### Other uses of visualize\n",
+    "The `visualize()` function can also depict non-issue images, labels or predictions alone, or just the image itself. Let's explore this with a few images in our dataset.\n",
+    "\n",
+    "We can save a visualization to file via the `save_path` argument. Note the label quality score is high for this example and it is marked as a non-issue. The given and predicted labels closely resemble each other contributing to the high score."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7e770d23",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_to_visualize = 0\n",
+    "image_path = IMAGE_PATH + labels[image_to_visualize]['seg_map']\n",
+    "print(image_path, '| idx', idx , '| label quality score:', scores[image_to_visualize], '| is issue:', image_to_visualize in issue_idx)\n",
```

Comment:
```suggestion
    "print(image_path, '| idx', image_to_visualize , '| label quality score:', scores[image_to_visualize], '| is issue:', image_to_visualize in issue_idx)\n",
```

`idx` is not defined, I believe you meant to pass in this `image_to_visualize` var instead? Please make sure the notebook fully executes with no errors after edits since var names are easily missed after making changes